src/app/onboarding/onboarding.module.ts

```diff
@@ -1,23 +1,45 @@
-import { NgModule } from '@angular/core';
+import {
```

Comment:
```suggestion
/* @format */
import {
```

Since this file has been run through Prettier, let's add the pragma to check it.